raiden/messages.py

```diff
@@ -489,16 +492,41 @@ class MediatedTransfer(LockedTransfer):
 
     cmdid = messages.MEDIATEDTRANSFER
 
-    def __init__(self, identifier, nonce, token, transferred_amount, recipient,
-                 locksroot, lock, target, initiator, fee=0):
+    def __init__(
+            self,
+            identifier,
+            nonce,
+            token,
+            transferred_amount,
+            recipient,
+            locksroot,
+            lock,
+            target,
+            initiator,
+            fee=0):
 
-        if nonce >= 2 ** 64:
+        if 0 > nonce >= 2 ** 64:
```

Comment:
Perhaps here also change the `ValueError` message to signify that nonce is either too large or negative?